kolibri/core/assets/src/views/CorePage/ImmersivePage.vue

```diff
@@ -1,32 +1,36 @@
 <template>
 
-  <div :style="wrapperStyles">
-    <ImmersiveToolbar
-      ref="appBar"
-      :appBarTitle="(!loading ? appBarTitle : '')"
-      :route="route"
-      :icon="icon"
-      :isFullscreen="primary"
-    />
-    <slot></slot>
-    <KLinearLoader
-      v-if="loading"
-      class="loader"
-      type="indeterminate"
-      :delay="false"
-    />
+  <div class="main">
```

Comment:
It also works just with: 
```Vue
  <div :style="wrapperStyles">
    <ScrollingHeader :scrollPosition="0">
      <ImmersiveToolbar
        ref="appBar"
        :appBarTitle="(!loading ? appBarTitle : '')"
        :route="route"
        :icon="icon"
        :isFullscreen="primary"
      />
      <KLinearLoader
        v-if="loading"
        type="indeterminate"
        :delay="false"
      />
    </ScrollingHeader>
    <slot></slot>
  </div>
```
But I added the `main` and `main-wrapper` classes for consistency with the `AppBarPage` component.